tests/models.ts

```diff
@@ -4,6 +4,7 @@ export class Model {
   public BooleanProp?: boolean;
   public ArrayProp?: Array<number | string | boolean | any[]>;
   public ObjProp?: Model2;
+  public ObjArrayProp?: Array<Model2>;
```

Comment:
minor: some inconsistency here between `Array<>` and `[]`. Maybe should be `Array<any>` instead of `any[]`?